core/src/docs/rfcs/2133_introduce_append_api.md

```diff
@@ -0,0 +1,100 @@
+- Proposal Name: `introduce_append_api`
+- Start Date: 2023-04-26
+- RFC PR: https://github.com/apache/incubator-opendal/pull/2133
+- Tracking Issue: 
+
+# Summary
+
+Separate append operations from the write operation.
```

Comment:
This is an addition instead of a `seperate`. `write` has different sementic with `append`.